readme.md

```diff
@@ -39,7 +39,7 @@ $ npm install --save react-keyframes
 **`<Keyframes { component = 'span', delay = 0, loop = 1, onStart, onEnd } />`**
 
 - Use `import { Keyframes } from 'react-keyframes'` or `require('react-keyframes').Keyframes`.
-- The `component` prop specifies what component `Keyframes` renders as.
+- The `component` prop specifies what component `Keyframes` renders as. Passing "Fragment" will render the component inside a `<React.Fragment />`. If no component is passed, the default is a "span" tag.
```

Comment:
It seems we can pass a React component to `React.createElement` so that `<Keyframes component={React.Fragment}>` just work. Maybe we should make it clear on README.

Reply:
I wonder if we should support a "Fragment" string too. What do you think?

Reply:
Ah! I didn't realize that. I updated the docs.

Also, I don't think we need to? I removed it from my code because doing `<Keyframes component={React.Fragment}>` seems to be sufficient. What do you think?